samples/Client/DaprClient/Program.cs

```diff
@@ -209,6 +212,34 @@ internal static async Task InvokeBalanceServiceOperationAsync(DaprClient client)
             Console.WriteLine($"Received balance {res.Balance}");
         }
 
+        internal static async Task InvokeThrowExceptionOperationAsync(DaprClient client)
+        {
+            Console.WriteLine("Invoking ThrowException");
+            var data = new { id = "17", amount = (decimal)10, };
+
+            HTTPExtension httpExtension = new HTTPExtension()
+            {
+                Verb = HTTPVerb.Post
+            };
+
+            try
+            {
+                // Invokes a POST method named "Withdraw" that takes input of type "Transaction" as define in the RoutingSample.            
+                await client.InvokeMethodAsync<object>("controller", "throwException", data, httpExtension);
```

Comment:
minor nitpick - you can omit `<object>` here - someone who's reading this will likely be confused about why the generic type here doesn't match `data`'s declared type.

Reply:
removed